pom.xml

```diff
@@ -71,7 +71,7 @@
         <commons-io.version>2.8.0</commons-io.version>
         <commons-lang.version>2.6</commons-lang.version>
         <integration.mysql.db.image>quay.io/eclipse/che--centos--mysql-57-centos7:latest-e08ee4d43b7356607685b69bde6335e27cf20c020f345b6c6c59400183882764</integration.mysql.db.image>
-        <integration.postgresql.db.image>quay.io/eclipse/che--centos--postgresql-96-centos7:9.6-b681d78125361519180a6ac05242c296f8906c11eab7e207b5ca9a89b6344392</integration.postgresql.db.image>
+        <integration.postgresql.db.image>centos/postgresql-13-centos7:1</integration.postgresql.db.image>
```

Comment:
would you prefer to use a copy of a specific build, in the `quay.io/eclipse/che--centos--postgresql...` space? 

That way you won't get unexpected updates. 

Also we likely need a downstream issue for this to switch to https://catalog.redhat.com/software/containers/rhel8/postgresql-13/5ffdbdef73a65398111b8362?container-tabs=overview&gti-tabs=registry-tokens

Reply:
I think that if we bump to a newer version, we should bump to a multi-arch image.

Else there is no urgency to move to this centos7 image

Reply:
https://quay.io/repository/eclipse/che--centos--postgresql-13-centos7?tag=1-71b24684d64da46f960682cc4216222a7e4ed8b1a31dd5a865b3e71afdea20d2&tab=tags created but it appears to be single-arch.

Reply:
how about we use https://quay.io/repository/eclipse/registry.redhat.io--rhel8--postgresql-13?tag=1-8755650687f2c94d7f0d08f491afa0cba97199ceb7946afed260a619a3f5f49b&tab=tags ? it's got x, arm, z, and p.

Reply:
@xbaran4 could you update the image to `quay.io/eclipse/che--centos--postgresql-13-centos7:1-71b24684d64da46f960682cc4216222a7e4ed8b1a31dd5a865b3e71afdea20d2`